### ISSUE TITLE
how to set ignored label for train and eval in object detection api

### ISSUE THREAD

**Issue by kingstarcraft** [tensorflow/models#4699]

**Comment by informatrix** [comment]
Have you tried filter out the location_gt, classes_gt inside trainer.py?

**Comment by kingstarcraft** [comment]
@informatrix thank you very much.I will try and set ignored_labe_wieght to zero for loss.

**Comment by bitfort** [comment]
I will recommend you touch base with http://stackoverflow.com/questions/tagged/tensorflow
, this is a great starting point to get going with debugging this issue -- feel free to post here if you discover a specific problem or bug.

I have also added someone from object detection in case they have something more to add.

**Comment by kingstarcraft** [comment]
the groundtruth_weights was used in model_main.py.
